### PR TITLE
Unity project vol2

### PR DESCRIPTION
**Pull request by AgaMazur**
Changed name of project

### CODE REVIEW

File: Podwodny świat tabliczki mnożenia/.collabignore [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# ===========================
```
Comment (ani119) [inline Podwodny świat tabliczki mnożenia/.collabignore:1]: As I wrote at your last pull request, do you see this filename? It includes polish chars and spaces :(

File: Podwodny świat tabliczki mnożenia/Assets/Food icons Pack/PNG/Materials/fish.mat [hunk 2]
```diff
@@ -0,0 +1,27 @@
+%YAML 1.1
```
Comment (ani119) [inline Podwodny świat tabliczki mnożenia/Assets/Food icons Pack/PNG/Materials/fish.mat:1]: all filenames includes polish chars and spaces :(

**Review by ani119 — CHANGES_REQUESTED** [review]
As I wrote at comments filenames don't change so I propose to delete all project and all branches and all pull request. Next create branch after that create project in the same folder which you have as git repository on your local disk. Add all changes and create commit, push to origin and finaly create one pull request

**Comment by ani119** [comment]
@AgaMazur I assigned you to issue #2 so let's create branch with name #2-add-basic-unity-project and the title / summary of commit should start with #2 too.

**Comment by AgaMazur** [comment]
Moved to #16.

**Pull request closed without merging**